tests/sentry/tsdb/test_redis.py

```diff
@@ -35,11 +35,11 @@ def tearDown(self):
             client.flushdb()
 
     def test_make_counter_key(self):
-        result = self.db.make_counter_key(TSDBModel.project, 1368889980, 1)
-        assert result == 'ts:1:1368889980:1'
+        result = self.db.make_counter_key(TSDBModel.project, 1, to_datetime(1368889980), 1)
+        assert result == ('ts:1:1368889980:1', 1)
 
-        result = self.db.make_counter_key(TSDBModel.project, 1368889980, 'foo')
-        assert result == 'ts:1:1368889980:33'
+        result = self.db.make_counter_key(TSDBModel.project, 1, to_datetime(1368889980), 'foo')
+        assert result == ('ts:1:1368889980:46', self.db.get_model_key('foo'))
```

Comment:
This behavior has changed at this method level but is the same overall result. This used to have [the output of the `get_model_key` provided](https://github.com/getsentry/sentry/blame/c1ce6511ac2de168cb7f60300a2cd5bf3b1723d6/src/sentry/tsdb/redis.py#L219-L224) to generate the hash key, now it's the [*input* to the `get_model_key` function which is called internally](https://github.com/getsentry/sentry/blame/15d8cde319c64d79b4759b81c028da81468e9ba6/src/sentry/tsdb/redis.py#L131).